### PR TITLE
Setup mapstruct into backend

### PR DESCRIPTION
**Pull request by kouamschekina**
- Setup Mapstruct into Backend
- Implemented a test for the mapstruct

### CODE REVIEW

**Review by Elwizzy12 — APPROVED** [review]
great job!

**Review by NkwaTambe — APPROVED** [review]

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/domain/model/Person.java
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/domain/model/Person.java:1]: Do we need this in our business logic? If yes, what for?

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/dto/PersonDto.java
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/dto/PersonDto.java:1]: Do we need this in our business logic? If yes, what for?

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/utils/PersonMapper.java
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/utils/PersonMapper.java:1]: Do we need this in our business logic? If yes, what for?

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/domain/model/Person.java
Comment (Elwizzy12) [reply power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/domain/model/Person.java:1]: The Person class does not appear to be part of the core business logic. It is simply a sample implementation and may not be strictly required. The existing logic is functioning correctly with the current sample. So, depending on the specific requirements and design of this application, we can easily incorporate the Person class, data transfer object, and associated mappings by making the necessary modifications to fit the application's needs.

File: power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonMapper.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.adorsys.gis.powerpay.powerpaybackend;
+
+// PersonMapper.java
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
```
Comment (stephane-segning) [inline power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonMapper.java:8]: Add the type component to this mapper, as we're working with Spring Boot.

https://www.baeldung.com/mapstruct#Injection

**Review by stephane-segning — CHANGES_REQUESTED** [review]
One last change:

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**